### ISSUE TITLE
Follow up to 'Error when adding standard Data Type 'List View - Members''

### ISSUE THREAD

**Issue by LordCererbus** [umbraco/Umbraco-CMS#12743]
### Which *exact* Umbraco version are you using? For example: 9.0.1 - don't just write v9

10.0.1

### Bug summary

Hi,

I recently posted a bug I found but my explanation of how to find it led to the wrong reproduction steps.

https://github.com/umbraco/Umbraco-CMS/issues/12714

I replied to it with a video of how to reproduce but I haven't heard anything so just in case you don't get notifications of replies to closed bug issues figured I'd re-raise it with a video showing how to reproduce.

I have tried this with fresh installs of 10.0.1 and 10.1-rc2 and the problem exists in both. This time I've attached a video showing exactly how to reproduce the issue.

**The Bug**
When adding a new 'List View - Members' property to an existing Data Template I am getting a popup error:

An error occurred
Ordering by username not supported.

Exception Details
System.NotSupportedException, System.Private.CoreLib, Version=6.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e: Ordering by username not supported.

### Steps to reproduce

1. Using a clean build of v10.0.1, go to Settings and add a Document Type (I used Document with Template).
2. Give the document template a name and save it.
3. Now add a new property, give it a name and select the editor type 'List View - Members'
4. Hit submit and as soon as you do it will error.

**Please note that you must save the document type first before adding the member list view.** If you add the member list view and then save the document type for the first time it seems to work.
The problem I've got is that I need to add it to an already existing document type.

https://user-images.githubusercontent.com/52680822/181243368-d8b1de66-64fc-4f9f-95a0-b2d647290d07.mp4

### Expected result / actual result

Hitting 'Submit' should add the property to the data template without a popup error.

---
_This item has been added to our backlog [AB#21091](https://dev.azure.com/umbraco/243e7927-03b2-44e2-908f-d4ac7ea5daaa/_workitems/edit/21091)_

**Comment by nul800sebastiaan** [comment]
Hey there @LordCererbus, when I read this issue I thought: "well, that's not good but sounds like an easy fix, I'll ask the team to handle it". When I talked to @kjac about it yesterday while he was working on it, we couldn't actually figure out what you were trying to do here.

Could you please explain what the point of having a list of members would have on a content item? We could understand it if you were trying to pick members for example (a member picker does exist) but we couldn't quite figure out why you'd need to list all members on content of a certain type. Specifically, the member list view was not designed to live in the content section, it needs context from the members section (which is why it doesn't work and throws an error). 

Maybe we're missing something, let us know!

**Comment by LordCererbus** [comment]
Hi, @nul800sebastiaan, thanks for looking into it.

I'm on the novice to intermediate skills end of Umbraco and this was my first time implementing Member features and I'll admit, you're right I was looking to implement a multi-member picker. I made the mistake of assuming that the member list view feature was the multi-picker I was after.
A couple of days ago I realised it wasn't when trying to work around the bug and just created a member picker in a nested list component.

After realising this, while I no longer need that issue fixing for myself to continue, I figured I'd leave the ticket open in case you'll wanted to take a stab at fixing what I'd found anyway.

Feel free to close this or keep it open for bug reference. There is no longer any urgency for it from me.

Thanks again for taking a look.
